tests/framework/OutStreams/line_plot.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" ?>
+<Simulation verbosity = 'debug'>
+  <RunInfo>
+    <WorkingDir>linePlot</WorkingDir>
+    <Sequence>generate_data,plot</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <TestInfo>
```

Comment:
can you move this block to the top of the file?

Reply:
moved.